Sources/BraintreeShopperInsights/BTShopperInsightsClientV2.swift

```diff
@@ -101,4 +101,23 @@ public class BTShopperInsightsClientV2 {
     public func isVenmoAppInstalled() -> Bool {
         application.isVenmoAppInstalled()
     }
+    
+    /// Generates customer recommendations
+    /// - Warning: This method is currently in beta and may change or be removed in future releases.
+    public func generateCustomerRecommendations(
```

Comment:
Can we move this up with the other create/update methods in the file? Also I think these should be using async/await vs completion.